### ISSUE TITLE
Cannot install because outdated version? but home assistant is up to date

### ISSUE THREAD

**Issue by rampestamper** [thomasloven/lovelace-layout-card#278]
My Home Assistant version: 2022.XX.X
2024.2.5
Layout-card version (FROM BROWSER CONSOLE):
-
What I am doing:
trying to install Layout-Card
What I expected to happen:
installing
What happened instead:
You are running Home Assistant version '2024.2.5', but this repository requires minimum version '2024.3.0b0' to be installed.
**Minimal** steps to reproduce:
tryed to update home assistant (docker) but this is the latest version currently available
```yaml
# The least ammount of code possible to reproduce my error

# End of code
```

Error messages from the browser console:
You are running Home Assistant version '2024.2.5', but this repository requires minimum version '2024.3.0b0' to be installed.
---

**By putting an X in the boxes ([X]) below, I indicate that I:**

- [] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).
- [] Have made sure I am using the latest version of the plugin.
- [] Have followed the troubleshooting steps of the "Common Problems" section of https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins.
- [] Understand that leaving one or more boxes unticked or failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by Taomyn** [comment]
I'm surprised to see a release that has a requirement for a pre-release when the full release is not even available.

**Comment by Arturfrain** [comment]
Same problem

**Comment by pavelkrato** [comment]
Same problem, sad.

**Comment by Arturfrain** [comment]
> Same problem, sad.

Yes sad especially when you built a lit of cards with this layout card.

**Comment by TheFes** [comment]
This version fixes an issue with the beta (regarding the new `selections`) view mode for dashboards.
There is no need to update to this version if you are not running the 2024.3 beta. Probably that's why it has the beta as minimum HA Core requirement.

**Comment by rampestamper** [comment]
> This version fixes an issue with the beta (regarding the new `selections`) view mode for dashboards.
> There is no need to update to this version if you are not running the 2024.3 beta. Probably that's why it has the beta as minimum HA Core requirement.

Ok but when you want to install this on a fresh ha install then it's not possible to install the layout-card. As I described in the post

**Comment by Taomyn** [comment]
> Ok but when you want to install this on a fresh ha install then it's not possible to install the layout-card. As I described in the post

Correct, and the fix should have been itself classed as a beta and used the beta mechanism in HACS to allow deployment for only those affected.

**Comment by rampestamper** [comment]
> @rampestamper @Taomyn in HACS you can select a version to install
> Without HACS you can download the files attached to the previous version, and put them in your `custom_components` folder

How can you select that when I select download I cannot select another version

**Comment by TheFes** [comment]
Hmm, looks like that's only possible after you installed it (under the Redownload option).
Not sure if it is possible to immediately download an older version then, but you can always use the custom installation option then.

**Comment by somar12** [comment]
Hi, same issue. Any suggestion to get success on installation?

**Comment by heibertelf** [comment]
> you can select a version to install

This feature has already been removed in one of the updates.

**Comment by pacemaker82** [comment]
Chose to skip version until this gets fixed

**Comment by SitusKiss** [comment]
> to skip version until this gets fixed

How does one go about this?

**Comment by SitusKiss** [comment]
> Hmm, looks like that's only possible after you installed it (under the Redownload option). Not sure if it is possible to immediately download an older version then, but you can always use the custom installation option then.

How does one go about this?

**Comment by TheFes** [comment]
@SitusKiss download the prior release and put the files manually in your custom_components folder

**Comment by Arturfrain** [comment]
In the meantime it should work with the new HA core 2024.3.0 released today
